gradle/ge.gradle

```diff
@@ -27,6 +31,25 @@ gradleEnterprise {
         obfuscation {
             ipAddresses { addresses -> addresses.collect { address -> "0.0.0.0"} }
         }
+
+        tag("JDK" + System.getProperty("java.version").find(/\d+/)) // major version
+        value("Java Vendor", System.getProperty("java.vendor"))
+        value("Java Version", System.getProperty("java.version"))
+
+        // Jenkins job name, less redundant "Solr" parts
+        String jenkinsJobName = System.getenv("JOB_NAME")
+        if (jenkinsJobName) { // looks like "Solr/Solr-Smoketest-9.5"
+            tag(jenkinsJobName.replaceAll(/Solr\W/,"")) // like "Smoketest-9.5"
+        }
+
+        // https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+        if (System.getenv("GITHUB_BASE_REF")) { // only exists for PRs
+            tag("PR")
+        }
+        String ghWorkflowName = System.getenv("GITHUB_WORKFLOW")
+        if (ghWorkflowName) {
+            tag(ghWorkflowName)
```

Comment:
The Common Custom User Data Gradle Plugin already adds this as a custom value called `CI workflow`

Reply:
Yes but I want a tag.  The Jenkins Job Name & GitHub workflow (similar) is a major build scan differentiator to quickly know what we're looking at.